editing.py

```diff
@@ -187,6 +187,8 @@ def add_area(self, area, edit_note, auto=False):
             "edit-area.iso_3166_1.0": area["ISO_3166_1"],
             "edit-area.iso_3166_2.0": area["ISO_3166_2"],
             "edit-area.iso_3166_3.0": area["ISO_3166_3"],
+            "edit-area.url.0.text": area["ext_links"],
```

Comment:
You're going to need to add two external links anyway, so it would make sense for this to support any number of links, probably passed as an array of objects? (or well, a list of dictionaries, I guess :) ).

I guess the same is true of ISO codes technically, but you're unlikely to need more than one in practice, so that one is less important.

Reply:
Sounds reasonable :)

Will add it ASAP in a couple of days